mantle/platform/api/azure/instance.go

```diff
@@ -55,8 +55,11 @@ func (a *API) getVMParameters(name, userdata, sshkey, storageAccountURI string,
 	//             from the following: Contains an uppercase character, Contains a
 	//             lowercase character, Contains a numeric digit, Contains a special
 	//             character) Control characters are not allowed"
-	password := fmt.Sprintf("%s%s%s", "ABC&", strconv.Itoa(rand.Int()), "xyz")
-
+	n, err := rand.Int(rand.Reader, big.NewInt(1<<63-1))
```

Comment:
I gave bad advice before.  Because `big.NewInt` takes an `int64`, `math.MaxInt64` would probably be better.

Reply:
fixed.